joomla/joomla-cms
# Add iframe lazyloading to the core wrapper feature

### Summary of Changes

Add iframe lazyloading to the core wrapper feature

### Testing Instructions

Create an iframe wrapper via the component (example: https://www.youtube.com/embed/tgbNymZ7vqY)
Create an iframe wrapper with the module
Check the html code and make sure the loading attribute is set (loading=lazy)
Disable the lazy loading option
Check the html code and make sure the loading attribute is set (loading=eager)

### Actual result BEFORE applying this Pull Request

There is no option to set the lazyloading attribute

### Expected result AFTER applying this Pull Request

There is a option to set the lazyloading attribute

![image](https://user-images.githubusercontent.com/2596554/110174901-2a057580-7e01-11eb-937e-b874e0d7439f.png)
![image](https://user-images.githubusercontent.com/2596554/110175088-7b156980-7e01-11eb-9197-ea6cee9db58d.png)

cc @bembelimen
